pkg/storeuploader/storeuploader.go

```diff
@@ -93,14 +93,14 @@ func (u *StoreUploader) Store(ctx context.Context, file, bucket, prefix string,
 			return err
 		}
 		if !exists {
-			logger.Infow("Creating bucket")
+			logger.Info("Creating bucket")
 			if err := u.client.MakeBucket(ctx, bucket, minio.MakeBucketOptions{}); err != nil {
 				return err
 			}
 		}
 	}
 
-	objectName := fmt.Sprintf("%s-%s-%s-%s", prefix, prefixSeparator, time.Now().Format("2006-01-02T15:04:05"), path.Base(file))
+	objectName := fmt.Sprintf("%s-%s-%s-%s", prefix, prefixSeparator, time.Now().Format("2006-01-02T150405"), path.Base(file))
```

Comment:
If we change the object name here, don't we need to change it in the restore produce in `etcd-launcher` as well? I was looking at that but I couldn't find a reference to the generated object name, so I'm a bit confused how the restore works.

Reply:
The only place I found where the etcdrestore is taking its name from is https://github.com/kubermatic/kubermatic/blob/master/pkg/handler/v2/etcdrestore/etcdrestore.go#L325, so the dashboard is sending us the name of the backup.

![screenshot-2022-06-24-141456](https://user-images.githubusercontent.com/127499/175532025-4b27d69a-9877-4b2f-a43d-b0ba591da9d7.png)

The list of backups seems to be created based on the EtcdBackupConfig status, so as long as there are names with .db in the status, they should show up in the dashboard and be transmitted 1:1 to the backend. So ...... we're fine?